system/lib/libc/musl/include/setjmp.h

```diff
@@ -22,10 +22,11 @@ typedef jmp_buf sigjmp_buf;
 /* XXX EMSCRIPTEN: No signals support, alias sigsetjmp and siglongjmp to their non-signals counterparts. */
 #if __EMSCRIPTEN__
 #define sigsetjmp(buf, x) setjmp((buf))
+#define siglongjmp(buf, val) longjmp(buf, val)
 #else
 int sigsetjmp (sigjmp_buf, int);
+int siglongjmp (sigjmp_buf, int);
```

Comment:
Keep the `_Noreturn` when moving this line?

Reply:
Done